.github/workflows/spellcheck.yml

```diff
@@ -0,0 +1,16 @@
+name: Spellcheck
+on:
+  push:
+    branches: [master]
+  pull_request:
+    branches: [master]
```

Comment:
No need for version branches?

Reply:
Added branches.

I don't think we should have a make target - CI will tell us, fairly quickly. If we see it's necessary, let's do it then, but IMHO we shouldn't.